pom.xml

```diff
@@ -298,6 +300,18 @@ limitations under the License.
                         <goals>deploy</goals>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-pmd-plugin</artifactId>
+                    <version>${org.apache.maven.plugins-maven-pmd-plugin.version}</version>
+                    <configuration>
+                        <rulesets>
+                            <ruleset>${session.executionRootDirectory}/pmd-rules.xml</ruleset>
+                        </rulesets>
+                        <excludes>
+                            <exclude>com/ericsson/bss/cassandra/ecaudit/auth/cache/</exclude>
```

Comment:
Let's remember to remove this exclude when we merge this into master (3.11) branch.

Reply:
ok